File: cs/dry.cs
class Program
{
	static void DoStuff()
	{
		System.Console.WriteLine("Doing stuff!");
	}
	
	static int DoSum(int a, int b)
	{
		return a + b;
	}
	
	public static void Main()
	{
		DoStuff();
		System.Console.WriteLine("2 + 1 = " + DoSum(2, 1));
	}
}

File: src/vb/dry.vb
Public Module Program
    Sub DoStuff()
        System.Console.WriteLine ("Doing stuff!")
    End Sub

    Function DoSum(ByVal a As Integer, ByVal b As Integer)
        DoSum = a + b
    End Function
   
    Sub Main()
        DoStuff
        System.Console.WriteLine($"2 + 1 = {DoSum(2, 1)}")
    End Sub
End Module